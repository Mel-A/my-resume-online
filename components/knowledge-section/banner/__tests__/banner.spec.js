import { shallow } from 'enzyme'
import Banner from '..'

jest.mock('react-i18next', () => ({
  useTranslation: () => ({t: key => key})
}))

test('banner renders correctly', () => {
  const banner = shallow(<Banner />)
  expect(banner).toMatchSnapshot()
})