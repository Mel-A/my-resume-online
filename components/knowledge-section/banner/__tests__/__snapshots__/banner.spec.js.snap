// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`banner banner renders correctly 1`] = `ShallowWrapper {}`;

exports[`banner banner renders theme correctly 1`] = `ReactWrapper {}`;
